service/gateway/router.go

```diff
@@ -59,6 +61,16 @@ func (g *Gateway) registerHandler(r *mux.Router) {
 		Path("/{object:.+}").
 		Queries(model.UploadProgressQuery, "").
 		HandlerFunc(g.queryUploadProgressHandler)
+	hostBucketRouter.NewRoute().
+		Name(getBucketAndPaymentRouterName).
+		Methods(http.MethodGet).
+		Path("/bucket/payment").
+		HandlerFunc(g.getBucketWithPaymentHandler)
```

Comment:
What happens if objectName=`/bucket/payment`?
It is best to modify the router and supplement the corresponding unit tests. It's easy to go wrong here.